### ISSUE TITLE
Migrate zio-delegate

### ISSUE THREAD

**Issue by ioleo** [zio/zio-macros#12]
Related to [issue](https://github.com/zio/zio-delegate/issues/4).

**Comment by ioleo** [comment]
Note to self: always crossCompile with overriden `isSnapshot := false` to check before you release.
```
[error] bad option: '-opt:l:inline'
[error] (coreJS / Compile / compileIncremental) Compilation failed
[error] Total time: 24 s, completed Oct 9, 2019 6:13:49 AM
```

**Comment by ioleo** [comment]
Released in `v0.3.0`.
